object/user.go

```diff
@@ -200,6 +202,17 @@ type ManagedAccount struct {
 	SigninUrl   string `xorm:"varchar(200)" json:"signinUrl"`
 }
 
+func (u *User) checkPasswordChangeRequiestAllowed() error {
+	if !u.isPasswordChangeRequiestAllowed() && u.PasswordChangeRequired {
+		return fmt.Errorf("PasswordChangeRequired not allowed for user '%s' due to be external one", u.Name)
```

Comment:
message can't understand

Reply:
so wait answer on 
https://github.com/casdoor/casdoor/pull/2104#discussion_r1271374338
and after can change error message or remove

Reply:
The error msg is so hard to understand, change it to a more understandable way

Reply:
how about ?
"PasswordChangeRequired = true - Does't supported for users from LDAP or Keycloak"

Reply:
PasswordChangeRequired is not supported to be enabled for users from LDAP or Keycloak

Reply:
ok fixed